knowledge-base/shivammathur/setup-php/action-security.yml

```diff
@@ -0,0 +1,116 @@
+name: 'setup-php'
+github-token:
+  environment-variable-name: COMPOSER_TOKEN
+  permissions:
+    contents: read
+    contents-reason: to access matching refs GitHub API, clone the repos, and get release assets.
+outbound-endpoints:
+  api.github.com:443:
+    port: 443
+    reason: to call GitHub API
```

Comment:
Can you please add specific GitHub APIs called? e.g. is it same as access matching refs GitHub API, clone the repos, and get release assets?

Reply:
setup-php calls matching refs API, but also sets this token in Composer's config which calls other API with this token to avoid rate limiting.

Reply:
setup-php calls matching refs API, but also sets this token in Composer's config which calls other API with this token to avoid rate limiting.